AirCasting/Utils/Strings.swift

```diff
@@ -305,300 +439,501 @@ struct Strings {
+                                                                          comment: "")
+    }
+    
     enum DefaultDeleteSessionViewModel {
-        static let all: String = LocalizedStringKey("All").stringKey
+        static let all: String = NSLocalizedString("All",
+                                                   comment: "")
     }
-
+    
     enum DeviceHandler {
-        static let alertTitle = LocalizedStringKey("Not supported device").stringKey
-        static let alertMessage = LocalizedStringKey("To use the AirBeam3 in standalone mode and sync the SD card, an iPhone8 or higher is required").stringKey
+        static let alertTitle = NSLocalizedString("Not supported device",
+                                                  comment: "")
+        static let alertMessage = NSLocalizedString("To use the AirBeam3 in standalone mode and sync the SD card, an iPhone8 or higher is required",
+                                                    comment: "")
     }
-
+    
     enum NetworkAlert {
-        static let alertTitle: String = LocalizedStringKey("No internet connection").stringKey
-        static let alertMessage: String = LocalizedStringKey("You need to have internet connection to continue").stringKey
+        static let alertTitle: String = NSLocalizedString("No internet connection",
+                                                          comment: "")
+        static let alertMessage: String = NSLocalizedString("You need to have internet connection to continue",
+                                                            comment: "")
     }
-
+    
     enum MicrophoneAlert {
-        static let title: String = LocalizedStringKey("Allow AirCasting to record audio").stringKey
-        static let message: String = LocalizedStringKey("To record sound level measurements, the app needs access to your microhpone.").stringKey
+        static let title: String = NSLocalizedString("Allow AirCasting to record audio",
+                                                     comment: "")
+        static let message: String = NSLocalizedString("To record sound level measurements, the app needs access to your microhpone.",
+                                                       comment: "")
     }
-
+    
     enum InAppAlerts {
-        static let assertError: String = LocalizedStringKey("Unsupported button count! For SwiftUI implementation max of 2 buttons is supported").stringKey
-        static let unableToLogOutTitle: String = LocalizedStringKey("No internet connection").stringKey
-        static let unableToLogOutMessage: String = LocalizedStringKey("To sign out, you must be connected to the Internet.").stringKey
-        static let unableToLogOutButton: String = LocalizedStringKey("Got it!").stringKey
-    }
-
+        static let assertError: String = NSLocalizedString("Unsupported button count! For SwiftUI implementation max of 2 buttons is supported",
```

Comment:
we don't really want to localize strings we use internally for asserts. This file should only contain user-facing strings